src/__tests__/__snapshots__/LabelGroup.js.snap

```diff
@@ -58,19 +56,19 @@ exports[`BranchName matches snapshot 1`] = `
   className="c0"
 >
   <span
-    className="c1"
+    className="c1 LabelItem"
     size="medium"
   >
     Default label
   </span>
   <span
-    className="c1"
+    className="c1 LabelItem"
```

Comment:
🤔 Why did `LabelItem` get added in this PR? I'd expect it to have been added in the 14.4.0 tracking branch but should already be on master

Reply:
Not too concerned about the snapshot changes though, they reflect what the accurate snapshots should look like